src/IntervalArithmetic.jl

```diff
@@ -80,7 +81,7 @@ export
 
 export showfull
 
-import Base: rounding, setrounding, setprecision
+import Base: setprecision
```

Comment:
Since we are getting rid of `rounding` and `setrounding`, shouldn't we amend (in parallel) the [docs](https://juliaintervals.github.io/pages/tutorials/tutorialArithmetic/index.html#advanced_options_precision_and_rounding_modes)? Similarly, we should add the corresponding docs of `IntervalArithmetic.set_directed_rounding`